plugins/modules/helm.py

```diff
@@ -139,6 +139,12 @@
     type: bool
     default: False
     version_added: "1.2.0"
+  history_max:
+    description:
+      - Limit the maximum number of revisions saved per release.
+    type: int
+    default: False
+    version_added: "2.1.2"
```

Comment:
```suggestion
    version_added: "2.2.0"
```

This will go in minor release rather than in patch release.